### DISCUSSION TITLE
vercel部署的网站无法正常访问

### DISCUSSION THREAD

**Discussion by YanSu5001** [DIYgod/RSSHub#11185] [Q&A] [answered]
如题，大概两个月前，vercel网站还能正常访问，而vercel部署的rsshub网页无法正常访问了，这是什么问题，是被防火墙了吗？

**Comment by sunwgit** [comment, marked as answer]
> 如题，大概两个月前，vercel网站还能正常访问，而vercel部署的rsshub网页无法正常访问了，这是什么问题，是被防火墙了吗？

被墙了

**Comment by QSlotus** [comment]
> 如题，大概两个月前，vercel网站还能正常访问，而vercel部署的rsshub网页无法正常访问了，这是什么问题，是被防火墙了吗？

你可以通过绑定自己的域名实现正常访问。
